### ISSUE TITLE
Windows Install error-options is not defined

### ISSUE THREAD

**Issue by alterhu2020** [nuxt-community/nuxt7#54]
When install the nuxt7 from fresh Window 10 with Nodejs v11.10.1 it shows below installation error:

![image](https://user-images.githubusercontent.com/5029046/54085504-de3aa580-4379-11e9-8b76-140924e57fbf.png)

cannot init the nuxt7 project.

**Comment by alterhu2020** [comment]
Also windows run the command `npm run dev` will trail the path incorrect: see below error:
![image](https://user-images.githubusercontent.com/5029046/54086447-848ba880-4384-11e9-9876-de796ee40f26.png)

you can see that the windows path had not parsed correct for the `\node_modules`.... or other

**Comment by alterhu2020** [comment]
@pi0 , this issue is critical in  the code  in `options.js`:

```
  // Icons
  if (options.f7Icons) {
    options.f7IconsSrc = resolvePath('fonts/framework7-icons.css')
  }

  if (options.mdIcons) {
    options.mdIconsSrc = resolvePath('fonts/material-icons.css')
  }


```

**Comment by alterhu2020** [comment]
Why the Mac machine throws the same exception as below:

![image](https://user-images.githubusercontent.com/5029046/54194908-4911eb00-44f8-11e9-870c-9507e09af5c1.png)

**Comment by pi0** [comment]
@alterhu2020 It seems you are using NPX which is not fully supported by nuxt. Would you please try using a normal `yarn` install?

**Comment by alterhu2020** [comment]
Hi,
@pi0 , It's not related to the `yarn` package issue, I had also tried using below yarn command in windows platform, it also throws the exceptions:
``
cd template
yarn
yarn run dev
```
![image](https://user-images.githubusercontent.com/5029046/54250992-ca14c500-4580-11e9-9849-937f091243cb.png)

**Comment by alterhu2020** [comment]
also  `npx sao nuxt-community/nuxt7 nuxt7-app` command is from your official README document, why it not works? thanks very much for your reply @pi0

**Comment by alterhu2020** [comment]
Hi @pi0 
Could you change the code in `options.js` for the import package using `hard code ` path instead of `resolve` method to get the icon path ,like below changes:
# lib\options.js
```

 // Icons
  if (options.f7Icons) {
    options.f7IconsSrc = 'nuxt7/lib/fonts/framework7-icons.css'
  }

  if (options.mdIcons) {
    options.mdIconsSrc = 'nuxt7/lib/fonts/material-icons.css'
  }

````
in this way it can support for windows platform as well, thanks .

**Comment by alterhu2020** [comment]
Close it as pull request #58

**Comment by pi0** [comment]
released in v3.1.0
